Cleaning DB/3 Reports cleaning.sql

```diff
@@ -0,0 +1,23 @@
+use CleaningDB 
+go
+
+select DeepCleans = 
+    concat(c.LastName, ', ', c.FirstName, ' (', c.PhoneNum, ') - ', ((c.RatePerHour * 1.5) * (c.AvgTime * 2))) 
+from Customer c
+
+select CustomerInfo = concat(c.LastName, ' - ', c.Frequency), NumOfMonths = datediff(month, c.StartDate, getdate())
+from Customer c 
+where c.EndDate is null
+order by NumOfMonths desc
+
+select AvgRatePerHourRegClean = avg(c.RatePerHour), AvgRatePerHourDeepClean = avg(c.RatePerHour * 1.5)
+from Customer c
+
+select PricePerYear =(c.RatePerHour * 52), * 
```

Comment:
The report asks that you only show the customer that the client earns the most from in a year

Reply:
Comment remains to be fixed